__tests__/app.test.js

```diff
@@ -25,213 +27,230 @@ describe("GET /api", () => {
+      });
   });
 });
 
-describe('PATCH /api/articles/:article_id', () => {
-  test('PATCH:201 updates the vote count of a specified article', () => {
+describe("PATCH /api/articles/:article_id", () => {
+  test("PATCH:201 updates the vote count of a specified article", () => {
     const newVote = {
-      inc_votes: 2
+      inc_votes: 2,
     };
     return request(app)
-      .patch('/api/articles/1').send(newVote)
+      .patch("/api/articles/1")
+      .send(newVote)
       .expect(201)
       .then((response) => {
         expect(response.body.result.votes).toBe(102);
       });
   });
 
-  test('PATCH:201 updates the vote count of a specified article with a negative decrement', () => {
+  test("PATCH:201 updates the vote count of a specified article with a negative decrement", () => {
     const newVote = {
-      inc_votes: -102
+      inc_votes: -102,
     };
     return request(app)
-      .patch('/api/articles/1').send(newVote)
+      .patch("/api/articles/1")
+      .send(newVote)
       .expect(201)
       .then((response) => {
         expect(response.body.result.votes).toBe(-2);
       });
   });
 
-  test('POST:404 not found when article id does not exist', () => {
+  test("POST:404 not found when article id does not exist", () => {
     const newVote = {
-      inc_votes: 2
+      inc_votes: 2,
     };
     return request(app)
-      .patch('/api/articles/999').send(newVote)
-      .expect(404).then(( response ) => {
+      .patch("/api/articles/999")
+      .send(newVote)
+      .expect(404)
+      .then((response) => {
         console.log(response.body, "response");
         const { msg } = response.body;
-        expect(msg).toBe('Not found'); 
-      })
+        expect(msg).toBe("Not found");
+      });
   });
 
-  test('POST:400 bad reqest when article id is not a number', () => {
+  test("POST:400 bad reqest when article id is not a number", () => {
     const newVote = {
-      inc_votes: 2
+      inc_votes: 2,
     };
     return request(app)
-      .patch('/api/articles/banana').send(newVote)
-      .expect(400).then(( response ) => {
-        console.log(response.body, "response");
+      .patch("/api/articles/banana")
+      .send(newVote)
+      .expect(400)
+      .then((response) => {
+
         const { msg } = response.body;
-        expect(msg).toBe('Bad request'); 
-      })
+        expect(msg).toBe("Bad request");
+      });
   });
 });
 
-describe('PATCH /api/articles/:article_id', () => {
-  test('DELETE:204 removes comment by id', () => {
-    const newVote = {
-      inc_votes: 2
-    };
+describe("DELETE /api/comments/:comment_id", () => {
+  test("DELETE:204 removes comment by id", () => {
     return request(app)
-      .patch('/api/articles/1').send(newVote)
-      .expect(201)
+      .delete("/api/comments/1")
+      .expect(204)
       .then((response) => {
-        expect(response.body.result.votes).toBe(102);
+        // why is my response body an empty object?
```

Comment:
A delete endpoint will never have anything on the body by design - so just checking for the 204 is fine!